### PR TITLE
Remove Prettier

### LINKED ISSUE #31: Consider removing prettier
It forces some changes I personally don't find more readable.

Examples:

```diff
         {
-          $or: [
-            { failureType: null },
-            { failureType: { $exists: false } },
-            { failureType: { $ne: FailureType.Hard } },
-          ],
+          $or: [{ failureType: null }, { failureType: { $exists: false } }, { failureType: { $ne: FailureType.Hard } }],
         },
```

Or this:

```ts
    const handleErrors = async (error: Error) => {
      if (error instanceof NoMoreEntriesException) {
        logger.trace(error.message)
      } else if (error instanceof InvalidClaim) {
         await updateEntryFailureReason(error.ipfsHash, FailureType.Hard, error.failureReason)
      } else if (error instanceof IPFSError) {
         logger.warn(error)
         await updateEntryFailureReason(error.ipfsHash, FailureType.Soft, FailureReason.IPFS)
      }
    }
```

Changes to:

```ts
    const handleErrors = async (error: Error) => {
      if (error instanceof NoMoreEntriesException) logger.trace(error.message)
      else if (error instanceof InvalidClaim)
        await updateEntryFailureReason(error.ipfsHash, FailureType.Hard, error.failureReason)
      else if (error instanceof IPFSError) {
        logger.warn(error)
        await updateEntryFailureReason(error.ipfsHash, FailureType.Soft, FailureReason.IPFS)
      }
    }
```

### PR DESCRIPTION
**Pull request by lautarodragan**
Removing prettier.

Fixes #31 

### TODO
- [ ] Confirm correct packages were removed
- [ ] Test in node
- [ ] Test in frost-api
- [ ] Test in frost-web
- [ ] Test in explorer-web

### CODE REVIEW

**Comment by wzalazar** [comment]
Prettier has several auto fixes, I think we lose more remove it.

**Comment by lautarodragan** [comment]
@wzalazar do you know what those auto fixes are? If we have a list, we can look for TSLint alternatives

File: poet-tslint.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
     "rulesDirectory": ["tslint-plugin-prettier", "./rules"],
```
Comment (lautarodragan) [inline poet-tslint.json]: Shouldn't this be changed too?

**Comment by lautarodragan** [comment]
Superseded by https://github.com/poetapp/tslint-rules/pull/141

**Pull request closed without merging**